src/data/stig_multipliers.ts

```diff
@@ -1728,6 +1860,66 @@ const DPS_STIG_MULTIPLIERS = Object.freeze({
       }),
     ],
   },
+  Verne: {
+    T: [
+      new Multiplier({
+        active: true,
+        name: "Verne T: ice DMG",
+        type: Type.EleDealt,
+        value: 20,
+      }),
+      new Multiplier({
+        active: true,
+        name: "Verne T: charged attacks deal bonus total DMG",
+        type: Type.TdmDealt,
+        value: 40,
+      }),
+    ],
+    M: [
+      new Multiplier({
+        active: true,
+        name: "Verne M",
+        type: Type.TdmDealt,
+        value: 20,
+      }),
+      new Multiplier({
+        active: true,
+        name: "Verne M: charged attacks deal bonus ice DMG",
+        type: Type.EleDealt,
+        value: 30,
+      }),
+    ],
+    B: [
+      new Multiplier({
+        active: true,
+        name: "Verne B: basic & charged attacks deal bonus ice DMG",
+        type: Type.EleDealt,
+        value: 50,
+      }),
+    ],
+    "2 set": [
+      new Multiplier({
+        active: false,
+        name: "Verne 2 set: ultimate evasion in burst mode boosts total DMG",
+        type: Type.TdmDealt,
+        value: 30,
+      }),
+      new Multiplier({
+        active: true,
+        name: "Verne 2 set: charged attacks in burst mode boost ice DMG by 15%. 4 stacks max.",
+        type: Type.EleDealt,
+        value: 60,
+      }),
+    ],
+    "3 set": [
+      new Multiplier({
+        active: false,
```

Comment:
Seems okay to set this to active by default, I don't think too many people will use Verne on anything other than Nyx

Reply:
Incorporated in latest commit